content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -17,13 +17,154 @@ aliases:
 
 ## Lassie
 
-Lassie is a simple retrieval client for Filecoin. It finds and fetches your data over the best retrieval protocols available.
+Lassie is a simple retrieval client for Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command:
 
-### Prerequisites
+```shell
+lassie fetch <CID>
+```
+
+Lassie can also be used as a reference library to fetch data from Filecoin. In the absence of a single uber-protocol which meet all data transfer needs, Lassie was designed to use multiple protocols for data transfers.
+
+![Lassie Architecture](lassie_architecture.png)
+
+### Lassie HTTP daemon
```

Comment:
Let's not lead with this, it's not really the primary mode we want to encourage—in fact there's a good chance it'll be spun out into a separate tool. I wouldn't mind if this section is trimmed down some more too, maybe with a link into the Lassie repo for more info.

For the audience, the primary mode they'll want is the CLI. In #1833 I did a bunch of work on that mode which could be pulled in. In that PR I ignored the daemon because I don't think it's going to be as helpful for the majority of this audience.

Reply:
Yes. Plan is to redo this section to reduce the info here. I will link it to Lassie repo.